File: nMODm.c
/*
    @author Sarah Kidd (www.linkedin.com/in/sarahkidd/)
    Description of problem at: https://www.codeeval.com/open_challenges/62/
*/

#include <stdio.h>
#include <string.h>

#define LINE_SIZE 1024
int main(int argc, char **argv)
{
    int n, m, divNum, remainder = 0;
	FILE *ifp = fopen(argv[1], "r");
    while(fscanf(ifp, "%d,%d\n", &n, &m) != EOF) {
        divNum = n/m;
        remainder = n - (divNum * m);
		printf("%d\n", remainder);
    }
    return 0;
}

File: 062/nMODm.js
/*
	@author Sarah Kidd (www.linkedin.com/in/sarahkidd/)
	Description of problem at: https://www.codeeval.com/open_challenges/62/
*/

var fs = require("fs");
fs.readFileSync(process.argv[2]).toString().split('\n').forEach(function (line) {
    if (line !== "") {
        var arr = line.split(",");
        var n = parseInt(arr[0]);
        var m = parseInt(arr[1]);
        var divNum = Math.floor(n / m);
        var remainder = n - (divNum * m);
        console.log(remainder);
    }
});